internal/keyctl/keyring.go

```diff
@@ -11,6 +11,8 @@
 package keyctl
```

Comment:
(Just so that it isn’t forgotten, we can’t _right now_ move the package to `internal/…`, unless we want the next release to be 4.0.0.)

Reply:
Right, that's why I was saying "before the next major bump" :^)